account/src/use-case/creatUserAccount.js

```diff
@@ -0,0 +1,21 @@
+let contas = []
+
+let dataAtualFormatada = new Date().toISOString().substring(0,10)
```

Comment:
Essa data eu recomendo colocar dentro da função, por que uma vez que este arquivo for lido pelo node, essa data não vai mudar mais a não ser que manualmente você faça essa mudança.